products/libllbuild/CMakeLists.txt

```diff
@@ -57,14 +57,6 @@ install(TARGETS libllbuild
   RUNTIME DESTINATION bin
   COMPONENT libllbuild)
```

Comment:
I think that we should nix this part as well.  Although it doesn't really matter much for Windows distributions as the packaging of the installer will drop it, it does help keep things clean.  Does it make sense to continue distributing the headers?  Or should we also be dropping those?  Sorry about this mess I created!

Reply:
I don't know if the rest is needed or not, haven't looked into it.